editor/editor_command_palette.cpp

```diff
@@ -251,8 +251,15 @@ void EditorCommandPalette::_add_command(String p_command_name, String p_key_name
 void EditorCommandPalette::execute_command(String &p_command_key) {
 	ERR_FAIL_COND_MSG(!commands.has(p_command_key), p_command_key + " not found.");
 	commands[p_command_key].last_used = OS::get_singleton()->get_unix_time();
-	commands[p_command_key].callable.call_deferred();
 	_save_history();
+
+	Callable::CallError ce;
+	const Callable &callable = commands[p_command_key].callable;
+	callable.callp(nullptr, 0, Variant(), ce);
```

Comment:
I think the return value, a.k.a `Variant()` needs to be declared above and reference same as the error.

```suggestion
	Callable::CallError ce;
	Variant ret;
	const Callable &callable = commands[p_command_key].callable;
	callable.callp(nullptr, 0, ret, ce);
```